fix(curriculum): update focus style instructions and add outline test in job application form lab (#67055)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/lab-job-application-form/66faac4139dbbd5dd632c860.md

You should add a `:focus` pseudo-class to the `input` and `textarea` elements to change their border color when focused. Use a list selector in the given order.

```js
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('input:focus, textarea:focus').getPropertyValue('border-color'));
```

> You should add a `:focus` pseudo-class to the `input` and `textarea` elements to remove the default outline when focused.

Reviewer: ```suggestion
You should use the selector for the `:focus` pseudo-class for `input` and `textarea` to also remove the default outline when focused.
```
